### PR TITLE
changed header color

### PR DESCRIPTION
**Pull request by vuonga1103**
Screen Shot 2020-11-02 at 9 34 06 PM

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -137,7 +138,8 @@ export default function Header() {
       return menuHeaders
         .filter(({ href }) => href !== '/logout' && href !== '/account')
         .map(({ id, href, label }) => {
-          let color = label === 'Join Us!' ? '#50E3C2' : 'white';
+          // Change "Join Us!" to another color
```
Comment (cfernando415) [inline src/components/Header/Header.js:141]: remove the comment

**Review by cfernando415 — CHANGES_REQUESTED** [review]
please see comment :-)

File: src/components/Header/Header.js (hunk 1 above)
Comment (vuonga1103) [reply src/components/Header/Header.js]: done! d2659116d3b701c0fb650123af1253b4667c77fe

**Review by cfernando415 — APPROVED** [review]
👍

**Pull request merged**